angular/Kanban/Swimlanes.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title:  Swimlanes
-description: Swimlanes
+title: Swimlanes in Angular Kanaban Control | Syncfusion
```

Comment:
Control name is Kanban, change and commit

Reply:
modified